pkg/virtctl/sev/attestation_test.go

```diff
@@ -15,143 +16,127 @@ import (
 	"kubevirt.io/client-go/api"
 	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/kubevirt/pkg/virtctl/sev"
-	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/clientcmd"
 )
 
 var _ = Describe("SEV Attestation", func() {
 	const vmiName = "testvm"
-	var vm *v1.VirtualMachine
 	var vmi *v1.VirtualMachineInstance
-	var vmInterface *kubecli.MockVirtualMachineInterface
 	var vmiInterface *kubecli.MockVirtualMachineInstanceInterface
 	var ctrl *gomock.Controller
 	var workDir string
-	var err error
 
 	BeforeEach(func() {
-		vm = kubecli.NewMinimalVM(vmiName)
-		vmi = api.NewMinimalVMI(vmiName)
 		ctrl = gomock.NewController(GinkgoT())
-		workDir, err = ioutil.TempDir("", "sev-test")
-		Expect(err).To(BeNil())
 		kubecli.GetKubevirtClientFromClientConfig = kubecli.GetMockKubevirtClientFromClientConfig
 		kubecli.MockKubevirtClientInstance = kubecli.NewMockKubevirtClient(ctrl)
-		vmInterface = kubecli.NewMockVirtualMachineInterface(ctrl)
 		vmiInterface = kubecli.NewMockVirtualMachineInstanceInterface(ctrl)
-		kubecli.MockKubevirtClientInstance.EXPECT().VirtualMachineInstance(k8smetav1.NamespaceDefault).Return(vmiInterface).AnyTimes()
-
-		vmiInterface.EXPECT().Get(vmi.Name, gomock.Any()).Return(vmi, nil).AnyTimes()
-		vmInterface.EXPECT().Get(vmi.Name, gomock.Any()).Return(vm, nil).AnyTimes()
-	})
-
-	AfterEach(func() {
-		if workDir != "" {
-			err = os.RemoveAll(workDir)
-			Expect(err).ToNot(HaveOccurred())
-			workDir = ""
-		}
+		vmi = api.NewMinimalVMI(vmiName)
+		workDir = GinkgoT().TempDir()
```

Comment:
oh I see here the GinkgoT().TempDir :) , maybe could you rebase the fix in the original commit? It would make it easier for reviewers